packages/react-core/src/evm/hooks/async/contracts.ts

```diff
@@ -512,20 +512,22 @@ export function useContractEvents(
 export function useContractRead(
   contract: RequiredParam<ValidContractInstance>,
   functionName: RequiredParam<string>,
-  ...args: unknown[] | [...unknown[], CallOverrides]
+  args?: unknown[],
```

Comment:
can we get the types all the way up to this as well?

Reply:
I want to leave that for a separate PR b/c its much more involved (due to the overlap between ValidContractInstance types)